events/babylon_discord_break_3_1_event.txt

```diff
@@ -0,0 +1,689 @@
+﻿namespace = ideology_break_3_1_event
+# Ideological break for level 3 reached
+ideology_break_3_1_event.1 = {
+	type = country_event
+	title = ideology_break_3_1_event.1.t
+	desc = ideology_break_3_1_event.1.d
+	flavor = ideology_break_3_1_event.1.f
+	event_image = {
+		video = "gfx/event_pictures/unspecific_vandalized_storefront.bk2"
+	}
+	icon = "gfx/interface/icons/event_icons/event_fire.dds"
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	duration = 3
+	trigger = {
+		# Caused when a nation hits 300 on their cohesion level.
+	}
+	option = {
+		name = ideology_break_3_1_event.1.a
+		# Blame the workers
+		default_option = yes
+		random_list = {
+			33 = {
+				modifier = {
+					if = {
+						limit = {
+							ruler = {
+								has_trait = bandit
```

Comment:
has been removed from spawning xd, the bandit trait

Reply:
Yea, but some leaders still start with it..? I think?

Reply:
yes